core/src/lib/bsock.h

```diff
@@ -179,6 +179,9 @@ class BareosSocket {
   virtual void RestoreBlocking(int flags) = 0;
   virtual bool ConnectionReceivedTerminateSignal() = 0;
   // Returns: 1 if data available, 0 if timeout, -1 if error
+  static inline constexpr int DataAvailable = 1;
+  static inline constexpr int Timeout = 0;
+  static inline constexpr int Error = -1;
```

Comment:
if I'm not mistaken that `inline` is redundant, but of course not wrong.

Reply:
Ill have to check again, but at least there is a difference between `static` and `static inline`.  Not sure if `constexpr` makes those the same again.

Reply:
https://stackoverflow.com/a/54467409 global variables should be `inline constexpr` to fix odr related issues that might arise.   `static` here is just because they are defined inside the class scope.

Reply:
Seems like constexpr member variables are always inline.  Thanks for the hint!